reference/5.1/Microsoft.PowerShell.Core/About/about_Using.md

```diff
@@ -64,20 +81,58 @@ $hashfromstream = Get-FileHash -InputStream $memorystream `
 $hashfromstream.Hash.ToString()
 ```
 
-The following script assumes a module named 'CardGames' was loaded
-automatically.
+### Example 2 - Load classes from a script module
+
+In this example, we have a PowerShell script module named **CardGames** the
+defines the following classes:
 
-The following classes are defined in the module:
+- **CardGames.Deck**
+- **CardGames.Card**
 
-- **Deck**
-- **Card**
+The `using module` command imports the module. `Import-Module` and the
+`#requires` statement only import the module functions, aliases, and variables,
+as defined by the module. Classes are not imported. The `using module`
+statement imports the classes defined in the module.
```

Comment:
Can reword this? I think just the first sentence could be removed and it'll be clearer.

Reply:
The sentence is needed but it could be clearer.